framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -529,8 +529,7 @@ protected void addInformationFromPreconditions(
                                 false);
                 info.insertValue(expr, annotation);
             } catch (FlowExpressionParseException e) {
-                // report errors here
-                analysis.checker.report(e.getResult(), methodTree);
```

Comment:
>Similar code is further down in `processPostconditions` and `processConditionalPostconditions`.
Shouldn't those also be changed?

The BaseTypeVisitor parses preconditions at both the declaration of a method and at it uses.  In both cases, if parsing fails, an error is used.  The BaseTypeVisitor does not parse postconditions at uses of methods, so an error won't be issued if parsing fails.  If this check is added to the BaseTypeVisitor, then these other two error reports can be removed.  (In this case, only type systems that use flow can benefit from postcondition annotations.)